corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/CoNLLMASCReader.java

```diff
@@ -0,0 +1,84 @@
+package edu.illinois.cs.cogcomp.nlp.corpusreaders;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import edu.illinois.cs.cogcomp.core.datastructures.IntPair;
+import edu.illinois.cs.cogcomp.core.datastructures.Pair;
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+
+/**
+ * Created by xle2 on 11/1/2017.
```

Comment:
Give a brief description of the corpus this reads (source, genre, annotation types)